conf/reef/rados/test-confs/rados_config_parameters.ini

```diff
@@ -0,0 +1,41 @@
+#MSG version 2 parameters
```

Comment:
Please make sure that the below values would still be same for reef.

Reply:
I will create a separate test suite for MSGRV2, Mclock, and additional parameters.
Created the following jira ticket.
https://issues.redhat.com/browse/RHCEPHQE-9675